File: net/Azure.Test.PerfStress/NoOpTest.cs
using System.Threading;
using System.Threading.Tasks;

namespace Azure.Test.PerfStress
{
    public class NoOpTest : PerfStressTest<PerfStressOptions>
    {
        public NoOpTest(PerfStressOptions options) : base(options) { }

        public override void Run(CancellationToken cancellationToken)
        {
        }

        public override Task RunAsync(CancellationToken cancellationToken)
        {
            return Task.CompletedTask;
        }
    }
}

File: java/azure-perfstress/src/main/java/com/azure/perfstress/NoOpTest.java
package com.azure.perfstress;

import reactor.core.publisher.Mono;

public class NoOpTest extends PerfStressTest<PerfStressOptions> {
    public NoOpTest(PerfStressOptions options) {
        super(options);
    }

    @Override
    public void Run() {
    }

    @Override
    public Mono<Void> RunAsync() {
        return Mono.empty();
    }
}